rust/fastsim-core/src/imports.rs

```diff
@@ -1,5 +1,5 @@
-pub(crate) use anyhow::{anyhow, bail, ensure};
```

Comment:
@kylecarow , what was your motivation for this change?

Reply:
@calbaker I figured if we were using anyhow::Result, anyhow::bail!, anyhow::ensure!, anyhow::anyhow!, etc. we might as well fully qualify where they come from. Just changes how the code looks, really.

If you prefer it the other way that's fine too, but we use quite a lot of anyhow macros and stuff

Reply:
as for the bincode import change, that just clarifies where `serialize` and `deserialize` in `traits.rs` come from

Reply:
>  but we use quite a lot of anyhow macros and stuff

That's my very reason for wanting a shorthand.  For imports that are widely used, I tend to prefer that.  For imports that are used only sparsely, I tend to prefer full-path usage.  I'm not super firm on any of this.

Reply:
@kylecarow , where did we leave off here?